### PR TITLE
Add var-file support

### PR DESCRIPTION
**Pull request by rgreinho**
Types of changes
----------------

- New feature (non-breaking change which adds functionality)

Description
-----------

Add the ability to read the content of a var-file to create new
variables in a workspace.

Checklist:
----------

- [] I have updated the documentation accordingly
- [] I have updated the Changelog (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,13 +53,14 @@ tfe-cli variable create exisiting-workspace --var akey=a_value
 Update an existing variable in a specific workspace:
 
 ```bash
-tfe-cli variable create exisiting-workspace -f --var akey=a_nother_value
+tfe-cli variable create my-exisiting-workspace -f --var akey=a_nother_value
```
Comment (jeremygaither) [inline README.md]: `a_nother_value` should just be `another_value`

File: cmd/variable.go [hunk 2]
```diff
@@ -54,45 +58,80 @@ var variableCreateCmd = &cobra.Command{
...
-					}
-					if _, err := client.Variables.Update(context.Background(), workspace.ID, v.ID, options); err != nil {
-						log.Fatalf("Cannot update variable %q: %s.", v.Key, err)
-					}
-					break
+			//Check if the variable already exists.
+			v, exists := indexedVars[*(opts.Key)]
+
+			// Update an existing variable.
+			if exists && force {
```
Comment (jeremygaither) [inline cmd/variable.go]: This is what I was talking about last time...

File: cmd/variable.go [hunk 3]
```diff
@@ -54,45 +58,80 @@ var variableCreateCmd = &cobra.Command{
...
+				if _, err := client.Variables.Update(context.Background(), workspace.ID, v.ID, options); err != nil {
+					log.Fatalf("Cannot update variable %q: %s.", *(opts.Key), err)
 				}
+				log.Debugf("Variable %q updated successfully.", *(opts.Key))
+				continue
+			}
+
+			// Skip an existing variable.
+			if exists {
+				log.Warningf("Cannot create %q: variable already exists.", *(opts.Key))
```
Comment (jeremygaither) [inline cmd/variable.go]: I think this should return an error to the user and exit with an error code, or they may assume everything was updated when it wasn't.

File: cmd/variable.go (hunk 2 above)
Comment (ss-remygreinhofer) [reply cmd/variable.go]: It is the same logic, I just made it clearer 😉

File: cmd/variable.go (hunk 3 above)
Comment (ss-remygreinhofer) [reply cmd/variable.go]: That makes sense. I'll change it.

File: README.md (hunk 1 above)
Comment (ss-remygreinhofer) [reply README.md]: addressed ✅

**Pull request merged**